### PR TITLE
Introducing MVP of wwan MBIM support

### PR DESCRIPTION
**Pull request by rvs**
This is pretty barebones still and involves a lot of screen scraping from mbimcli. I propose we leave it as-is for now, and allow implementation to get tested a bit. Then, if we still like it, we can re-write it more in the style of 0001-produce-json-output.patch

### CODE REVIEW

File: pkg/wwan/usr/bin/wwan-init.sh [hunk 1]
```diff
@@ -115,32 +187,45 @@ reset_modem() {
...
-#        move this to the wwan's build.yml
-modprobe -a qcserial usb_wwan qmi_wwan cdc_wdm
+detect_mode() {
+  modprobe -a qcserial usb_wwan qmi_wwan cdc_wdm cdc_mbim cdc_acm
+  if [ "$(basename "$(readlink "/sys/class/usbmisc/$CDC_DEV/device/driver/module")")" = cdc_mbim ]; then
+     MODE=mbim
+  fi
+}
+
+# lets statrt with detecting what we're dealing with
```
Comment (eriknordmark) [inline pkg/wwan/usr/bin/wwan-init.sh:197]: Nit typo should be "start"

**Review by eriknordmark — APPROVED** [review]
LGTM

File: pkg/wwan/usr/bin/wwan-init.sh (hunk 1 above)
Comment (rvs) [reply pkg/wwan/usr/bin/wwan-init.sh:197]: LOL! Fixed!

File: pkg/wwan/usr/bin/wwan-init.sh [hunk 2]
```diff
@@ -1,11 +1,11 @@
 #!/bin/sh
-# set -x
+set -x
```
Comment (eriknordmark) [inline pkg/wwan/usr/bin/wwan-init.sh:2]: This is generating lots of logs at least on devices without a wwan interface. We should revert.

**Pull request merged**